core/src/main/java/com/sequenceiq/cloudbreak/converter/v4/stacks/database/DatabaseAvailabilityTypeToDatabaseResponseConverter.java

```diff
@@ -0,0 +1,18 @@
+package com.sequenceiq.cloudbreak.converter.v4.stacks.database;
+
+import org.springframework.stereotype.Component;
+
+import com.sequenceiq.cloudbreak.api.endpoint.v4.stacks.request.database.DatabaseAvailabilityType;
+import com.sequenceiq.cloudbreak.api.endpoint.v4.stacks.response.database.DatabaseResponse;
+import com.sequenceiq.cloudbreak.converter.AbstractConversionServiceAwareConverter;
+
+@Component
+public class DatabaseAvailabilityTypeToDatabaseResponseConverter  extends AbstractConversionServiceAwareConverter<DatabaseAvailabilityType, DatabaseResponse> {
```

Comment:
We are moving away from using Spring's conversions service, because its very hard to debug and follow through the code. Can you please implement it without extending `AbstractConversionServiceAwareConverter`?

Reply:
not now.